plugins/migration/test/rename.mocha.js

```diff
@@ -0,0 +1,527 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Tests for the rename script.
+ */
+
+import {assert} from 'chai';
+import {doRenamings} from '../bin/rename.js';
+
+
+suite('Rename', function() {
```

Comment:
I think this should be a pretty good start to a test suite. If you can think of any other cases that need to be covered I'm happy to add them.

Reply:
So far so good.  We've already thought of a few more and I'm sure will think of more yet, but a solid start!